travisci/python-linter_harness.sh

```diff
@@ -15,7 +15,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-PYTHON_SOURCE_LOCATIONS=('scripts')
+PYTHON_SOURCE_LOCATIONS=('scripts' 'src/python/lib/ensembl/compara')
```

Comment:
Don't we want to check the tests too ? (genuine question)

Reply:
I tried to mirror Perl's linter, but I keep forgetting that in Perl `t` is contained inside `modules`, but not in Python :man_facepalming:. Fixed.